R/RLearner_classif_glmboost.R

```diff
@@ -49,12 +54,17 @@ trainLearner.classif.glmboost = function(.learner, .task, .subset, .weights = NU
 predictLearner.classif.glmboost = function(.learner, .model, .newdata, ...) {
   type = ifelse(.learner$predict.type == "response", "class", "response")
   p = predict(.model$learner.model, newdata = .newdata, type = type, ...)
+  fam = getLearnerParVals(.learner)$family
   if (.learner$predict.type  == "prob") {
-    td = .model$task.desc
-    p = p[, 1L]
-    levs = c(td$negative, td$positive)
-    return(propVectorToMatrix(p, levs))
+    if (fam %in% c("AdaExp", "AUC")) {
+      stopf("prediction.type = 'prob' not implemented for family %s", fam)
+    } else {
+      td = .model$task.desc
```

Comment:
Well, I am not completely sure how to handle this: glmboost with family Binomal can predict probabilities (this is why the learner has the "prob" property) But if you switch family to AUC or AdaExp only classes can be predicted. This is why I implemented an error message.
Will add a note if throwing an error for this purpose is ok!